examples/block_jid.py

```diff
@@ -114,6 +116,11 @@ def jid(s):
             help="If given, prints the block list at the end of the operation",
         )
 
+        config_default = os.path.expanduser(
+            '~/.config/aioxmpp_credentials.ini')
```

Comment:
Well, many many years of experience persuaded me that mixing code (not mentioning Linux distro packages) and configuration is the path to hell. Configuration should be separate from the code, and ``$HOME`` (or ``~/.config``) is the place to put the configuration to, so why not to do it properly? I guess you are on Mac (where there is usually a way less of discipline), aren't you?